### ISSUE TITLE
problem with code blocks displaying

### ISSUE THREAD

**Issue by nufskiy** [gearmobile/gearmobile.github.io#5]
code blocks marked with backticks looks strange on page https://gearmobile.github.io/javascript/require-nodejs/

![issue](https://user-images.githubusercontent.com/77888616/205056339-e3faae95-936a-4bbe-aaf5-ec2b637e137e.png)

**Win 10, Chrome 107.0.5304.122**

**Comment by gearmobile** [comment]
yes indeed - this is a rather critical syntax problem, which has not yet been solved by any settings in the blog.
at the moment, the most effective way is to check all blog articles and manually correct such shortcomings.
